interfaces/handlerInterface.go

```diff
@@ -0,0 +1,21 @@
+package interfaces
+
+// These are the messages read off and written into the websocket. Since this
```

Comment:
comment on exported type Message should be of the form "Message ..." (with optional leading article)